EventPlanning.DAL/Activity.cs

```diff
@@ -12,7 +12,7 @@ public class Activity
         public string Description { get; set; }
         public int MinParticipants { get; set; }
         public int MaxParticipants { get; set; }
-        public virtual ICollection<EventActivity> EventActivities { get; set; }
+        public List<EventActivity> EventActivities { get; set; }
```

Comment:
As this is only EF related entities - why not make it `IEnumerable`? 
https://stackoverflow.com/questions/3628425/ienumerable-vs-list-what-to-use-how-do-they-work
This is why we would want to use `IEnumerable`